Detectors/EMCAL/workflow/src/RawToCellConverterSpec.cxx

```diff
@@ -87,6 +90,7 @@ void RawToCellConverterSpec::init(framework::InitContext& ctx)
 
 void RawToCellConverterSpec::run(framework::ProcessingContext& ctx)
 {
+  std::cout << "Called" << std::endl;
```

Comment:
idem, though this looks like debug message

Reply:
Ooops, yes, same as above. Will fix immediately